goal_src/game.gp

```diff
@@ -324,6 +335,15 @@
          "out/iso/6COMMON.TXT")
   )
 
+(defstep
+  :in '("game/assets/subtitle/jak1/game_subtitle_en.txt"
```

Comment:
an alternate way to set this up is to have a single `subtitle` file that then references all the languages files (kind of like how DGO is set up now).

Reply:
That works too, and I think I can come up with a way to only use 1 input file and also fix the `:arg` issue.